requirements.txt

```diff
@@ -1,3 +1 @@
-requests
-simplejson
-nose
```

Comment:
They're specified in tox. This is better because requirements specifies what's needed by the project as a whole, whereas tox specifies test- and python version-specific deps